api/src/operations/request/index.ts

```diff
@@ -16,8 +16,8 @@ export default defineOperationApi<Options>({
 			acc[header] = value;
 			return acc;
 		}, {} as Record<string, string>);
-
-		const result = await axios({ url, method, data: body, headers: customHeaders });
+		
+		const result = await axios({ encodeURIComponent(url), method, data: body, headers: customHeaders });
```

Comment:
It's not possible to use a function like this directly in an object without providing a key. When using a variable doing this will interpolate the variable as key like so `{ test }` is equal to ` { test: test }` however with a function this will result in a syntax error. 
```suggestion
		const result = await axios({ url: encodeURIComponent(url), method, data: body, headers: customHeaders });
```

Reply:
Ohh thanks @br41nslug , I forgot about this thing about function...Thanks for review, I will add it.

Reply:
Awesome, thanks for contributing 😃